src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -42,10 +42,15 @@ const processOptimoBlocks = pipe(
   addRecommendationsBlock,
 );
 
-const transformJson = async json => {
+// Here pathname is passed as a prop specifically for CPS includes
+// This will most likely change in issue #6784 so it is temporary for now
```

Comment:
Nice to see the comment here, re: above I would probably include a comment in both locations but maybe it would be neater to do it in one place what do you think?